### ISSUE TITLE
Errors for M1 Build & Universal Build

### ISSUE THREAD

**Issue by ChristopherA** [BlockchainCommons/GordianCosigner-iOS#15]
Using PR #4:

I do seem to be able to successfully build for "My Mac"or "My Mac (Rosetta)" using my MacBook Air (M1, 2020), but when it starts I get:

<img width="900" alt="image" src="https://user-images.githubusercontent.com/69103/104767879-bd83c980-5721-11eb-9d3e-6bc08fc8935a.png">

In Xcode I see:

```
Get failed: The specified item could not be found in the keychain.
Set failed: A required entitlement isn't present.
```

I see a different error and the build is NOT successful when I selected to build for "Any Mac (Apple Silicon, Intel)". I get this error:

<img width="254" alt="image" src="https://user-images.githubusercontent.com/69103/104767483-143cd380-5721-11eb-9b1b-fbfc9f87ecd7.png">

**Comment by ChristopherA** [comment]
My guess is that two different settings have to be fixed, one being the entitlement, and the other being some universal build setting that is not set properly.

**Comment by ChristopherA** [comment]
I'm now about to build for Simulator (iPod touch 7th & iPhone 12 max) and for "My Mac" (both m1 and rosetta), but I'm not able to build a "Universal" — I get the same "a build only device cannot be used to run this target". 

I'll close this issue with the merge, but I'm not sure about the "build only device" so I'll do it as a separate issue.

**Comment by ChristopherA** [comment]
Addressed in PR #4.
